pages/apis/rest_api/team_pipelines.md

```diff
@@ -0,0 +1,174 @@
+# Team pipelines API
+
+The team pipelines API allows users to review, create, update and destroy pipelines associated with a team in your organization.
+
+## Team pipeline data model
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>pipeline_id</code></th><td>UUID of the associated pipeline</td></tr>
```

Comment:
I think you can probably drop "associated" since it's just the pipeline in the model:

```suggestion
  <tr><th><code>pipeline_id</code></th><td>UUID of the pipeline</td></tr>
```